workflows/prognostic_c48_run/runtime/steppers/machine_learning.py

```diff
@@ -162,12 +163,29 @@ def predict(model: MultiModelAdapter, state: State) -> State:
     return {key: cast(xr.DataArray, output[key]) for key in output.data_vars}
 
 
+def _is_state_update_variable(key, state: State):
```

Comment:
Seems like this function should be defined near where `State` is defined or in `runtime.names`.